### PR TITLE
Fix migration issues and add simple model tests

### PR DESCRIPTION
**Pull request by pdbreen**

### CODE REVIEW

File: database/migrations/2015_01_01_100000_create_country_table.php [hunk 1]
```diff
@@ -8,7 +8,7 @@ class CreateCountryTable extends Migration
 {
     public function up()
     {
-        Schema::create('country', function (Blueprint $table) {
+        Schema::create('countries', function (Blueprint $table) {
```
Comment (pdbreen) [inline database/migrations/2015_01_01_100000_create_country_table.php:11]: I'm assuming this was a bug as well - that the table name should be plural like other tables.

Comment (drewroberts) [reply database/migrations/2015_01_01_100000_create_country_table.php:11]: Nice catch. Thanks!

**Comment by drewroberts** [comment]
I'm going to think about this a little more. I am leaning towards changing the `zip_codes` table to be `zips` so that we can use `zip_code` instead of `zip_code_code` as the reference to it since the primary field is `code` instead of `id` for that table.

**Comment by drewroberts** [comment]
Issue #17

File: database/factories/TimezoneFactory.php [hunk 2]
```diff
@@ -13,10 +13,12 @@ class TimezoneFactory extends Factory
...
     {
+        $timezone = $this->faker->unique()->timezone;
+
         return [
-            'name' => $this->faker->timezone,
-            'title' => $this->faker->timezone,
-            'php' => $this->faker->timezone,
+            'name' => $timezone,
+            'title' => $timezone,
+            'php' => $timezone,
```
Comment (drewroberts) [inline database/factories/TimezoneFactory.php:21]: This is a great fix. It was causing a lot of test failures.

**Review by drewroberts — APPROVED** [review]
Thank you! ✂️

**Pull request merged**